konflux-ci/ui/core/proxy/proxy.yaml

```diff
@@ -154,33 +131,51 @@ spec:
           readOnlyRootFilesystem: true
           runAsNonRoot: true
           runAsUser: 1001
+      - image: kennethreitz/httpbin
+        name: httpbin
+        resources:
+          limits:
+            cpu: 300m
+            memory: 256Mi
+          requests:
+            cpu: 30m
+            memory: 128Mi
       - image: quay.io/oauth2-proxy/oauth2-proxy@sha256:3da33b9670c67bd782277f99acadf7026f75b9507bfba2088eb2d497266ef7fc
         name: oauth2-proxy
+        env:
+        - name: OAUTH2_PROXY_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: oauth2-proxy-client-secret
+              key: client-secret
+        - name: OAUTH2_PROXY_COOKIE_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: oauth2-proxy-cookie-secret
+              key: cookie-secret
         args:
           - --provider
-          - keycloak-oidc
+          - oidc
+          - --provider-display-name
+          - "Dex OIDC"
+          - --client-id
+          - oauth2-proxy
           - --http-address
           - "127.0.0.1:6000"
+          - --redirect-url
+          - https://localhost:9443/oauth2/callback
           - --oidc-issuer-url
-          - https://localhost:9443/idp/realms/redhat-external
+          - https://localhost:9443/idp/
+          - --cookie-secure
+          - "false"
```

Comment:
we need to use a secure cookie that can't get accessed via javascript. It's important for security.

When using a secure cookie, you would need to also set the `--cookie-name` flag. see https://oauth2-proxy.github.io/oauth2-proxy/configuration/overview/

Reply:
Changed.